autoload/leaderf/python/leaderf/filetypeExpl.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import os.path
+from leaderf.utils import *
+from leaderf.explorer import *
+from leaderf.manager import *
+
+
+# *****************************************************
+# FiletypeExplorer
+# *****************************************************
+class FiletypeExplorer(Explorer):
+    def __init__(self):
+        pass
+
+    def getContent(self, *args, **kwargs):
+        result = [
+            os.path.basename(f).replace(".vim", "")
+            for f in lfEval("globpath(&rtp, 'syntax/*.vim')").split("\n")
+        ]
+
+        # to unique
```

Comment:
`globpath(&rtp, 'syntax/*.vim')` is a little slow, it takes about 0.7s on my machine. I think we can make a cache for the result. You can refer to https://github.com/Yggdroot/LeaderF/blob/master/autoload/leaderf/python/leaderf/helpExpl.py.

Reply:
@Yggdroot Thank you.
It seems better to create a cache.
I fixed.